test/jdk/tools/launcher/ExecutionEnvironment.java

```diff
@@ -24,49 +24,27 @@
 /*
  * @test
  * @bug 4780570 4731671 6354700 6367077 6670965 4882974
- * @summary Checks for LD_LIBRARY_PATH and execution  on *nixes
- * @requires os.family != "windows" & !vm.musl & os.family != "aix"
+ * @summary Checks for LD_LIBRARY_PATH and execution on *nixes
+ * @requires os.family != "windows"
  * @library /test/lib
  * @modules jdk.compiler
  *          jdk.zipfs
- * @compile -XDignore.symbol.file ExecutionEnvironment.java
- * @run main/othervm -DexpandedLdLibraryPath=false ExecutionEnvironment
- */
-
-/*
- * @test
- * @bug 4780570 4731671 6354700 6367077 6670965 4882974
- * @summary Checks for LD_LIBRARY_PATH and execution  on *nixes
- * @requires os.family == "aix" | vm.musl
- * @library /test/lib
- * @modules jdk.compiler
- *          jdk.zipfs
- * @compile -XDignore.symbol.file ExecutionEnvironment.java
- * @run main/othervm -DexpandedLdLibraryPath=true ExecutionEnvironment
+ * @run main/othervm ExecutionEnvironment
  */
 
 /*
  * This tests for various things as follows:
  * Ensures that:
- *   1. uneccessary execs do not occur
+ *   1. unneccessary execs do not occur
  *   2. the environment is pristine,  users environment variable wrt.
  *      LD_LIBRARY_PATH if set are not modified in any way.
  *   3. the correct vm is chosen with -server and -client options
- *   4. the VM on Solaris correctly interprets the LD_LIBRARY_PATH32
- *      and LD_LIBRARY_PATH64 variables if set by the user, ie.
- *      i. on 32 bit systems:
- *         a. if LD_LIBRARY_PATH32 is set it will override LD_LIBRARY_PATH
- *         b. LD_LIBRARY_PATH64 is ignored if set
- *      ii. on 64 bit systems:
- *            a. if LD_LIBRARY_PATH64 is set it will override LD_LIBRARY_PATH
- *            b. LD_LIBRARY_PATH32 is ignored if set
- *   5. no extra symlink exists on Solaris ie.
+ *   4. no extra symlink exists on Solaris i.e.
```

Comment:
You can remove the Solaris comment, we do not build/test on Solaris for some time .

Reply:
I think it is wrong to completely remove point 4. You should rather remove the term "on Solaris". The thing itself is still tested in testNoSymLink().

Reply:
> I think it is wrong to completely remove point 4. You should rather remove the term "on Solaris". The thing itself is still tested in testNoSymLink().

Yes just remove Solaris not the whole comment  (on the other hand, is the symlink point still valid/important these days) ?